src/query/storages/fuse/src/operations/gc.rs

```diff
@@ -305,7 +306,9 @@ impl FuseTable {
         // Purge segments&blocks by chunk size
         let segment_locations = Vec::from_iter(segments_to_be_purged);
         for chunk in segment_locations.chunks(chunk_size) {
-            let locations = self.get_block_locations(ctx.clone(), chunk, false).await?;
+            let locations = self
```

Comment:
```suggestion
            // since we are purging files, the ErrorCode::STORAGE_NOT_FOUND error can be safely ignored.
            let locations = self
```

add a line of doc, hope I get it right.